frontend/build.config.js

```diff
@@ -47,7 +47,8 @@ module.exports = {
    */
   vendor_files: {
     js: [
-      'vendor/angular/angular.js'
+      'vendor/angular/angular.js',
```

Comment:
Also in the `test_files: {}` section the entry for `'vendor/appengine-channel-api-stub/dist/channel-api-stub.js'` should be removed as it is not used anymore.

Reply:
I will remove